parcels/codegenerator.py

```diff
@@ -176,16 +176,18 @@ def __getattr__(self, attr):
 
 
 class StatusCodeNode(IntrinsicNode):
-    symbol_map = {'Success': 'SUCCESS', 'Evaluate': 'EVALUATE', 'Repeat': 'REPEAT', 'Delete': 'DELETE', 'StopExecution': 'STOP_EXECUTION',
+    symbol_map = {'Success': 'SUCCESS', 'Evaluate': 'EVALUATE',
```

Comment:
personally, I would have attached a small comment above that says 'Line 1 are StateCodes, Line 2 are OperationsCodes and Line 3 (and beyond) are ErrorCodes' - to avoid this nice line separation to disappear after just a few future commits.